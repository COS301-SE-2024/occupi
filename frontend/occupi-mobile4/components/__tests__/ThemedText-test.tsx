import React from 'react';
import renderer from 'react-test-renderer';
import { ThemedText } from '../ThemedText';

jest.mock('@/hooks/useThemeColor', () => ({
  useThemeColor: jest.fn().mockReturnValue('#000000'),
}));

describe('ThemedText', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ThemedText>Snapshot test!</ThemedText>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});