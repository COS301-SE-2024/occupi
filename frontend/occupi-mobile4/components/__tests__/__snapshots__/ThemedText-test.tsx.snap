// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThemedText renders correctly 1`] = `
<div
  className="css-text-146c3p1"
  dir="auto"
  style={
    {
      "color": "rgba(0,0,0,1.00)",
      "fontSize": "16px",
      "lineHeight": "24px",
    }
  }
>
  Snapshot test!
</div>
`;

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      {
        "fontSize": 16,
        "lineHeight": 24,
      },
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;
